source/docs/javascript_usage.md

```diff
@@ -4,6 +4,26 @@ title: Usage in Javascript
 Embark includes in your dapp the EmbarkJS library. This library abstracts different functionality so you can easily & quickly build powerful dapps that leverage different decentralized technologies.
 Embark will automatically initialize EmbarkJS with the configurations set for your particular environment.
 
+### Provider
+
+According to [EIP1102](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1102.md), Dapps MUST request account access and Embark allows you to choose how you want to deal with request access.
+
+In the smart contracts configuration you can find this snippets:
```

Comment:
Which file?

Reply:
Agree with @bgits here. We should be as explicit as possible, or at least refer to another documentation. In this case it makes sense to probably say something like:

> An Embark application's Smart Contract configuration file (typically located in `config/contracts.js` comes with a `dappAutoEnable` property which controls whether or not EmbarkJS should automatically try to request account access:

Reply:
Answer: config/contracts.js
=> I will add it